.github/workflows/publish.yml

```diff
@@ -21,4 +21,3 @@ jobs:
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
         with:
           command: publish
-          args: --dry-run
```

Comment:
I unfortunately can't comment to lines not changed, but:

1. Can you please pin actions by hash?
2. If you haven't done so, can you ensure that the environment requires approval before executing?

Reply:
I'll make another PR that fixes the hash.  The environment already requires approval.

Reply:
Good call. We require separate approval for execution so we're good there, but yeah we still need to update to use the action's hash.